src/features/CourtCaseList/CourtCaseList.tsx

```diff
@@ -35,34 +13,19 @@ interface Props {
 }
 
 const CourtCaseList: React.FC<Props> = ({ courtCases, order = "asc", currentUser }: Props) => {
-  const classes = useStyles()
   const { basePath, query } = useRouter()
-
-  let searchParams = new URLSearchParams(encode(query))
-  searchParams = deleteQueryParamsByName(["unlockException", "unlockTrigger"], searchParams)
-
   const orderByParams = (orderBy: string) => `${basePath}/?${new URLSearchParams({ ...query, orderBy, order })}`
-  const caseDetailsPath = (id: number) => `${basePath}/court-cases/${id}`
-  const unlockCaseWithReasonPath = (reason: "Trigger" | "Exception", caseId: string) => {
-    searchParams.append(`unlock${reason}`, caseId)
-    return `${basePath}/?${searchParams}`
-  }
-
-  const canUnlockCase = (lockedUsername: string): boolean => {
-    return currentUser.groups.includes("Supervisor") || currentUser.username === lockedUsername
-  }
-
   const tableHead = (
     <Table.Row>
       <Table.Cell></Table.Cell>
-      <Table.CellHeader>
+      <Table.CellHeader setWidth={"178px"}>
         <ColumnOrderIcons columnName={"defendantName"} currentOrder={query.order} orderBy={query.orderBy}>
           <Link href={orderByParams("defendantName")} id="defendant-name-sort">
             {"Defendant Name"}
           </Link>
         </ColumnOrderIcons>
       </Table.CellHeader>
-      <Table.CellHeader>
+      <Table.CellHeader setWidth={"115px"}>
```

Comment:
Why is it `115px` specifically? Would it be worth sticking it in a variable with a meaningful name for visibility?

Reply:
It is as per the design and is specific to this header so abstracting it into a global variable may not be helpful.

Reply:
I was thinking just a variable in this component, not a global one.